Sources/sourcekit-lsp/SourceKitLSP.swift

```diff
@@ -137,6 +150,18 @@ struct SourceKitLSP: ParsableCommand {
   )
   var indexPrefixMappings = [PathPrefixMapping]()
 
+  @Option(
+    name: .customLong("compilation-db-search-path", withSingleDash: false),
+    parsing: .unconditionalSingleValue,
```

Comment:
Any reason why you decided to choose `unconditionalSingleValue` here instead of the default `singleValue`?

Reply:
No strong reason, though now that I think about it `singleValue` might be better as --compilation-db-search-path --some-other-flag may have a better compilation error.

Reply:
That’s what I thought as well.